test/e2e/config.go

```diff
@@ -15,7 +15,7 @@ var (
 	healthcheck       = "quay.io/libpod/alpine_healthcheck:latest"
 	ImageCacheDir     = "/tmp/podman/imagecachedir"
 	fedoraToolbox     = "registry.fedoraproject.org/fedora-toolbox:36"
-	volumeTest        = "quay.io/libpod/volume-plugin-test-img:latest"
+	volumeTest        = "quay.io/luap99/volume-plugin-test-img:20220623"
```

Comment:
Done - but I took the liberty of changing `podman` in Makefile to **`bin/podman`** in case it makes a difference.
```console
$ bin/podman push quay.io/libpod/volume-plugin-test-img:20220623
Getting image source signatures
Copying blob eec3e6bf68a4 done  
Copying blob 4794e514b721 skipped: already exists  
Copying config 24bf7448fb done  
Writing manifest to image destination
Storing signatures
```